OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListActivity.java

```diff
@@ -22,10 +21,12 @@
 import java.util.ArrayList;
 
 import de.greenrobot.event.EventBus;
+import roboguice.inject.InjectView;
 
 public class MyCoursesListActivity extends BaseSingleFragmentActivity {
 
     @NonNull
+    @InjectView(R.id.coorinator_layout)
     private CoordinatorLayout coordinatorLayout;
```

Comment:
Note that you don't need to have a direct reference to the `CoordinatorLayout`. You can use any descendant view, and the Snackbar will find it by trawling through it's ancestors.